diff -ruN a/Python/main.py b/Python/main.py
--- a/Python/main.py	2024-08-26 17:33:56.168008547 +0200
+++ b/Python/main.py	2024-08-26 17:46:58.718922376 +0200
@@ -31,10 +31,10 @@
 import configparser
 
 if os.environ.get('container') == 'flatpak':
-    user_home_folder = os.path.expanduser('~')
-    log_folder = os.path.join(user_home_folder,'.var/app/io.github.julian_hochhaus.LG4X_V2/cache/Logs')
+    log_folder = ('.var/app/io.github.julian_hochhaus.LG4X_V2/cache/Logs')
     os.makedirs(log_folder, exist_ok=True)
-    log_file_path = os.path.join(user_home_folder, '.var/app/io.github.julian_hochhaus.LG4X_V2/cache/Logs/app.log')
+    log_file_path = ('.var/app/io.github.julian_hochhaus.LG4X_V2/cache/Logs/app.log')
+    config_file_path = ('/app/config/config.ini')
 else:
     script_directory = os.path.dirname(os.path.abspath(__file__))
     log_folder = os.path.join(script_directory, '../Logs')
