diff -ruN a/Python/main.py b/Python/main.py
--- a/Python/main.py	2023-09-12 18:49:01.271171333 +0200
+++ b/Python/main.py	2023-09-12 23:52:24.051284562 +0200
@@ -27,10 +27,18 @@
 import traceback  # error handling
 import logging  # error handling
 from logging.handlers import RotatingFileHandler
-script_directory = os.path.dirname(os.path.abspath(__file__))
-log_folder = os.path.join(script_directory, '../Logs')
-os.makedirs(log_folder, exist_ok=True)
-log_file_path = os.path.join(script_directory, '../Logs/app.log')
+
+if os.environ.get('container') == 'flatpak':
+    user_home_folder = os.path.expanduser('~')
+    log_folder = os.path.join(user_home_folder,'.var/app/io.github.julian_hochhaus.LG4X_V2/cache/Logs')
+    os.makedirs(log_folder, exist_ok=True)
+    log_file_path = os.path.join(user_home_folder, '.var/app/io.github.julian_hochhaus.LG4X_V2/cache/Logs/app.log')
+else:
+    script_directory = os.path.dirname(os.path.abspath(__file__))
+    log_folder = os.path.join(script_directory, '../Logs')
+    os.makedirs(log_folder, exist_ok=True)
+    log_file_path = os.path.join(script_directory, '../Logs/app.log')
+
 max_log_size = 4*1024*1024 # 4MB
 backup_count = 5  # Number of backup files to keep
 handler = RotatingFileHandler(log_file_path, maxBytes=max_log_size, backupCount=backup_count)
